extensions/wikia/UserPreferencesV2/UserPreferencesV2.i18n.php

```diff
@@ -584,27 +591,27 @@
 	'preferences-v2-date' => 'Format des dates :',
 	'preferences-v2-time' => 'Fuseau horaire :',
 	'preferences-v2-skin' => 'Disposition :',
-	'prefs-emailv2' => 'Courriel',
-	'prefs-email-me-v2' => 'M’envoyer un courriel quand…',
+	'prefs-emailv2' => 'E-mail',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "E-mail" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)